posts/postsRoutes.js

```diff
@@ -0,0 +1,102 @@
+const express = require('express');
+const router = express.Router();
+const db = require('../data/db');
+
+router.use(express.json());
+
+router.post('/', (req, res) => {
+  const { title, contents } = req.body;
+  if (!title || !contents) {
```

Comment:
Try and use && instead of || here as it will make sure both inputs are populated